Core1Platform/AdministratingCore1/05_EngineRoom.md

```diff
@@ -0,0 +1,239 @@
+# Engine room
+
+The engine room contains some dashlets with information on vCore utilization, database storage utilization, request and error statistics, and jobs currently running.
+
+
+
+## Open and close the engine room
+
+Access to the engine room is available in any workspace you have opened in the *Core1 Platform*.
+
+
+#### Prerequisites
+
+- You have administrator rights for the current instance.
+
+#### Procedure
+
+*Actindo Core1 Platform > Any workspace*
+ 
+   ![Core1 Platform](../../Assets/Screenshots/Core1Platform/Core1.png "[Core1 Platform]")
+
+1. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button center top.  
+   The engine room is opened with the current workspace overlapped. It contains four dashlets.
+
+   ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+2. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button again to close it.
+   The workspace is visible again.
+
+
+
+## Clear tenant cache
+
+Clearing the tenant cache removes all entries that are currently in the cache of the instance. It is used if there is a general issue in processing, and if changes are made directly through the database rather than through system logic. It is recommended to use this function only if it is really necessary. Clearing the cache will slow down the instance considerably. 
+
+
+#### Prerequisites
+
+- You have administrator rights for the current instance.
+
+#### Procedure
+  
+*Actindo Core1 Platform > Any workspace*
+
+![Core1 Platform](../../Assets/Screenshots/Core1Platform/Core1.png "[Core1 Platform]")
+
+
+1. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button center top.  
+   The engine room is opened. The current workspace is overlapped.
+
+   ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+2. Click the [CLEAR TENANT CACHE] button top right.   
+   The following warning message is displayed:   
+
+   ![Warning message](../../Assets/Screenshots/Core1Platform/AdministratingCore1/ClearCacheWarning.png "[Warning message]")
+
+3. Click the [ABBRECHEN] (Cancel) button if you want to cancel the action. 
+
+4. If desired, click the [FORTFAHREN] (Continue) button to clear the cache.   
+   The cache is cleared. Afterwards, it will be rebuilt again. The processes on the instance are significantly slowed down.
```

Comment:
Afterwards, it will be rebuilt again. The processes on the instance are significantly slowed down. -> Ist das wirklich nötig?

Reply:
ja